artiq/test/coredevice/test_moninj.py

```diff
@@ -1,7 +1,12 @@
 import unittest
 import asyncio
+import numpy
+from collections import defaultdict
+from contextlib import contextmanager
+from unittest import IsolatedAsyncioTestCase
```

Comment:
Not used anywhere else in ARTIQ (it didn't exist when most of those tests were written), Needs to be introduced consistently in a separate PR or not used at all.

Reply:
`IsolatedAsyncioTestCase` would allow the use of async in each test case function by doing the same thing. I just forgot to remove it, since we removed `async` support and used asyncio loop instead.

Reply:
removed, done